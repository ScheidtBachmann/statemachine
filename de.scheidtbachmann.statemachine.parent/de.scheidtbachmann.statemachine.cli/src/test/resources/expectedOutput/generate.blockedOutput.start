The provided output path does not exist, creation failed with an exception:
Reason: java.nio.file.AccessDeniedException:
