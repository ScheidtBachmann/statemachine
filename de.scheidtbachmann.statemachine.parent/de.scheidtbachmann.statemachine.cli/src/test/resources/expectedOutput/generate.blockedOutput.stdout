The provided output path does exist, but writing is not permitted.
